examples/nextjs/README.md

```diff
@@ -1,19 +1,25 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
-
 ## Getting Started
 
-First, run the development server:
+First, setup the appropriate environment variables in `.env.local`:
+
+- `NACELLE_SPACE_ID` - your Nacelle Space Id from the Nacelle dashboard
+- `NACELLE_GRAPHQL_TOKEN` - your Nacelle GraphQL Token from the Nacelle dashboard
+
+To use Contentful's Preview Mode, you'll need the following environment variables in addition to the two above:
+
+- `NACELLE_PREVIEW_MODE` - a boolean indicating if preview mode should be enabled. `true` will enable preview mode.
+- `VERCEL_GITHUB_COMMIT_REF` - a Vercel [system environment variable](https://vercel.com/docs/build-step#system-environment-variables) that specifies the branch the deployment was made from
```

Comment:
It might be useful to specify that this environment variable is set automatically by Vercel / only makes sense in the context of a Vercel deploy. One thing to keep in mind is that even though Vercel sets the value for `VERCEL_GITHUB_COMMIT_REF`, it still needs to be added to the list of environment variables:

<img width="826" alt="Screen Shot 2020-11-16 at 1 33 43 PM" src="https://user-images.githubusercontent.com/5732000/99293229-5aa0af80-2810-11eb-937b-2eea2116a07d.png">

Reply:
Thanks for taking a look! Added clarification for that variable